cocos/core/renderer/models/skeletal-animation-utils.ts

```diff
@@ -224,20 +224,16 @@ export class JointTexturePool {
         for (let i = 0; i < joints.length; i++) {
             const node = skinningRoot.getChildByPath(joints[i]);
             const bound = boneSpaceBounds[i];
-            if (!node) { // don't skip null `bound` here, or it becomes mesh-specific
-                if (buildTexture) { uploadJointData(textureBuffer, 12 * i, Mat4.IDENTITY, i === 0); }
-                continue;
-            }
-            getWorldTransformUntilRoot(node, skinningRoot, m4_1);
+            const mat = node ? getWorldTransformUntilRoot(node, skinningRoot, m4_1) : Mat4.IDENTITY;
             if (bound) {
-                aabb.transform(ab_1, bound, m4_1);
+                aabb.transform(ab_1, bound, mat);
                 ab_1.getBoundary(v3_3, v3_4);
                 Vec3.min(v3_min, v3_min, v3_3);
                 Vec3.max(v3_max, v3_max, v3_4);
             }
             if (buildTexture) {
-                Mat4.multiply(m4_1, m4_1, bindposes[i]);
-                uploadJointData(textureBuffer, 12 * i, m4_1, i === 0);
+                if (mat !== Mat4.IDENTITY) { Mat4.multiply(mat, mat, bindposes[i]); }
```

Comment:
This could modify the freezed object Mat4.IDENTITY. You should use another output matrix